turbinia/lib/utils.py

```diff
@@ -75,8 +75,8 @@ def extract_artifacts(artifact_names, disk_path, output_dir):
   # Plaso image_export expects artifact names as a comma separated string.
   artifacts = ','.join(artifact_names)
   image_export_cmd = [
-      'image_export.py', '--artifact_filters', artifacts, '--write', output_dir,
-      '--partitions', 'all', disk_path
+      'sudo', 'image_export.py', '--artifact_filters', artifacts, '--write',
```

Comment:
Woops, that's a mistake.
I didn't want to run everything as root and now I'm adding sudo's everywhere in anger

Reply:
From https://wiki.debian.org/SystemGroups
```
disk: Raw access to disks. Mostly equivalent to root access.
```

Reply:
So if we make turbinia user in the 'disk' group  we might as well run as root and be free of access issues.
If we run as disk, we still won't be able to access part of mounted file system that is not readable to a normal user.
I can try to mount -o uid=turbinia_uid  but I worry about other side effects... like when a file ends up being `chmod 000`,  root cat still read it but not the owner. This might be of an issue when some python code in a task tries to open a file to read it.

I'm not sure how to proceed

Reply:
I think that for now we are fine to *not* add the turbinia user to the disk group and rely more on sudo for now.  I don't really want to run as root, but one thing @berggren was working on was scoping the sudo privileges to only what was needed.  Going that route gives us a bit more restricted privileges (than even the disk group) and also allows us to know exactly where we need some type privileged access in the codebase so that in the future we can re-factor it out with a more intelligent execution wrapper (or something).  The one thing that doesn't solve is privileged access from python itself.  I can't think of any examples of where that happens today (LMK if you're aware of any), so maybe that's ok for now, and if it's needed we can re-factor that code into utilities that we can shell out to.  SG?

Reply:
I'm keeping some sudo's then =)

I *might* have issues later on with docker-explorer module, which will read json files and then do things.
I think mlegin also wanted to explore a mounted file system for binaries, most likely with python (or some image_export.py trick maybe?)